introduction.md

```diff
@@ -0,0 +1 @@
+Hi. My name is Maahd. I'm working as a software developer at a small startup in Germany. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.